Finjinon/PhotoCaptureViewController.swift

```diff
@@ -165,6 +173,16 @@ public class PhotoCaptureViewController: UIViewController, PhotoCollectionViewLa
                 self.previewView.alpha = 1.0
             }
         }
+
+        NSNotificationCenter.defaultCenter().addObserverForName(UIDeviceOrientationDidChangeNotification, object: nil, queue: nil) { (NSNotification) -> Void in
```

Comment:
`addObserverForName(_:object:queue:)` returns a token that must be invalidated when you no longer need the observer. Looks like you're not unsubscribing from notifications in deinit or elsewhere?